app/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,5 @@
     <color name="colorPrimary">#6200EE</color>
     <color name="colorPrimaryDark">#3700B3</color>
     <color name="colorAccent">#03DAC5</color>
+    <color name="colorMain">#16b13a</color>
```

Comment:
Đặt tên theo tên màu và conventions nhé 
https://chir.ag/projects/name-that-color/#16B13A
```suggestion
    <color name="color_green_malachite">#16b13a</color>
```